docs/bookmarks.md

```diff
@@ -0,0 +1,221 @@
+pass a bookmark's name to commands that want a revision as argument. For example,
+`jj new main` will create a new revision on top of the "main" bookmark. Use
+`jj bookmark list` to list bookmarks and `jj bookmark` to create, move, or delete
+bookmarks. There is currently no concept of an active/current/checked-out bookmark.
+
+Currently Jujutsu maps its Bookmarks to Git Branches and stores them as that 
+in the Git backend. This means that all Bookmarks will be reflected as 
+Git Branches, this may change in the future. 
+
+## Remotes and tracked bookmarks
+
+Jujutsu records the last seen position of a bookmark on each remote (just like
+Git's remote-tracking branches). This record is updated on every `jj git fetch`
+and `jj git push` of the bookmark. You can refer to the remembered remote bookmark
+positions with `<bookmark name>@<remote name>`, such as `jj new main@origin`. `jj`
+does not provide a way to manually edit these recorded positions.
+
+A remote bookmark can be associated with a local bookmark of the same name. This is
+called a **tracked remote bookmark**, which currently maps to a Git remote 
+branch. When you pull a tracked bookmark from a remote, any changes compared to
+the current record of the remote's state will be propagated to the corresponding
+local bookmark, which will be created if it doesn't exist already.
+
+!!! note "Details: how `fetch` pulls bookmarks"
+
+    Let's say you run `jj git fetch --remote origin` and, during the fetch, `jj`
+    determines that the remote's "main" bookmark has been moved so that its target is
+    now ahead of the local record in `main@origin`.
+
+    `jj` will then update `main@origin` to the new target. If `main@origin` is
+    **tracked**, `jj` will also apply the change to the local bookmark `main`. If the
+    local target has also been moved compared to `main@origin` (probably because you
+    ran `jj bookmark set main`), then the two updates will be merged. If one is ahead
+    of the other, then that target will become the new target. Otherwise, the local
+    bookmark will become conflicted (see the ["Conflicts" section](#conflicts) below
+    for details).
+
+Most commands don't show the tracked remote bookmark if it has the same target as
+the local bookmark. The local bookmark (without `@<remote name>`) is considered the
+bookmark's desired target. Consequently, if you want to update a bookmark on a
+remote, you first update the bookmark locally and then push the update to the
+remote. If a local bookmark also exists on some remote but points to a different
+target there, `jj log` will show the bookmark name with an asterisk suffix (e.g.
+`main*`). That is meant to remind you that you may want to push the bookmark to
+some remote.
+
+If you want to know the internals of bookmark tracking, consult the 
+[Design Doc][design].
+
+### Terminology summary
+
+- A **remote bookmark** is a bookmark ref on the remote. `jj` can find out its
```

Comment:
Assuming we're making this change, I'd revert the changes here and mention a "Git remote" somewhere in this sentence.